packages/fast-usdc/test/supports.ts

```diff
@@ -149,9 +155,11 @@ export const commonSetup = async (t: ExecutionContext<any>) => {
   });
   const timer = buildZoeManualTimer(t.log);
   const marshaller = makeFakeBoard().getReadonlyMarshaller();
-  const storage = makeFakeStorageKit('mockChainStorageRoot', {
-    sequence: false,
-  });
+  const storage = makeFakeStorageKit(
+    'fun', // Fast USDC Node
```

Comment:
Optional: Could we just name this fastUsdc? Is it intentionally different than the real node? Seeing "fun" in the test file was confusing at first.

Reply:
> Is it intentionally different than the real node

It is intentional, to show that the contract root is configured by the core-eval and not the contract itself.

I do think that's worth keeping. I chose a three letter one to keep the paths short and wrap lines less.